test/Table_Tests/src/Helpers/Table_Aggregate_Auto_Constructor.enso

```diff
@@ -0,0 +1,69 @@
+from Standard.Base import all
+from Standard.Table import all
+from Standard.Database import all
+from Standard.AWS import all
+from Standard.Google_Api import all
+import Standard.Visualization
+
+collapsed operator16985 =
+    operator58114 = operator16985.aggregate ['Organisation'] columns=[(..Count), ..Sum]
+    operator35300 = operator16985.aggregate ['Organisation'] columns=[(Aggregate_Column.Count), Aggregate_Column.Sum]
+    [operator58114, operator35300]
+
+perform print=IO.println =
+    row1 = [ 3 ]
+    row2 = [ 2 ]
+    row3 = [ 3 ]
+    operator16985 = Table.from_rows [ 'Organisation' ] [row1, row2, row3 ]
+
+    interesting_uuids = Vector.new_builder
+        # line 9: (..Count)
+        . append "4cf46843-236f-4551-a9cc-30dd7552bb5f" # 287, 1
```

Comment:
Comments in the 
- #8415

resulted in `NullPointerException` and required 9cc208dc2e1cd235ce24bc499bb8f2c235efea76 fix.